src/v3/src/util/formUtils.ts

```diff
@@ -106,7 +104,6 @@ const getPIVButtonElement = (
       type: ButtonType.BUTTON,
       step: IDX_STEP.PIV_IDP,
       dataSe: 'piv-card-button',
-      classes: `${piv?.className || ''} piv-button`,
```

Comment:
Since we wont be using this className on the piv object, we may want to mark this for deprecation in the public docs to not confuse consumers.

Reply:
Ticket for tracking: https://oktainc.atlassian.net/browse/OKTA-699701

Reply:
Can we append the piv className to the data-se attribute?